implementations/rust/ockam/ockam_api/src/nodes/registry.rs

```diff
@@ -122,6 +122,39 @@ impl KafkaServiceInfo {
         &self.kind
     }
 }
+
+#[derive(Debug, Clone)]
+pub(crate) struct ForwarderRegistryInfo {
```

Comment:
What's the benefit of using this struct over `RemoteForwarderInfo`? 

If there isn't I'd remove it and store the `RemoteForwarderInfo` in the registry directly.

Reply:
Fixed in latest commit - it was mostly there to be consistent with the other registry-exclusive structs like `InletInfo` and `OutletInfo`. These could be similarly refactored